cli/src/config.rs

```diff
@@ -105,6 +108,23 @@ impl Default for PriceFeedConfigWrapper {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct KolliderPriceFeedConfigWrapper {
+    #[serde(default = "bool_true")]
```

Comment:
Think this should probably be false by default. `#[serde(default)]` should work as the default for `bool` is `false`.

Reply:
I changed the default to false